docs/DEPRECATION_CHECKLIST.md

```diff
@@ -64,6 +62,13 @@ with the Release team.
 The deprecated code finally gets removed.
 Don’t forget to [announce this accordingly](#announce-the-deprecation-and-removal).
 
+## Update the crate's version
+
+Inline with [SemVer's guidelines](https://semver.org/#spec-item-4), update the major/minor version in `Cargo.toml`
+depending on the following:
+- bump the major version when the crate's version >= 1.0.0
+- bump the minor version when the crate's version < 1.0.0
```

Comment:
should this be first aligned with the current release process? it's probably part of the pipeline now, and should it be even done by a developer?

Reply:
Indeed. The ideal (IMO) is to have the dev mark the PR as a breaking change. 
The release eng then will bump the version in Cargo.toml and publish the crate on crates.io (possibly keeping it aligned with the version in `Cargo.toml` (now is not 🙄 )